docs/index.rst

```diff
@@ -123,25 +129,42 @@ and some of the core features of Read the Docs.
    /faq
 
 
-Step-by-step Guides
--------------------
+How-to Guides
+-------------
 
 These guides will help walk you through specific use cases
 related to Read the Docs itself, documentation tools like Sphinx and MkDocs
 and how to write successful documentation.
 
-* :doc:`/guides/tools`
-* :doc:`/guides/platform`
-* :doc:`/guides/commercial`
+* **For documentation authors**:
```

Comment:
This shows we need more of these guides 🤔

Reply:
Double yes: we need more guides, and some of the current ones should probably be moved to Sphinx...